tests/unit/core/components/TestSiteNotices.js

```diff
@@ -82,4 +84,14 @@ describe(__filename, () => {
 
     expect(root.find(Notice)).toHaveLength(2);
   });
+
+  it('renders a notice when user has been logged out', () => {
```

Comment:
Should we also have a test that proves that this notice is not shown if the user has not been logged out?